Funcky/Sequence/Return.cs

```diff
@@ -5,6 +5,8 @@ namespace Funcky
 {
     public static partial class Sequence
     {
-        public static IEnumerable<TItem> Return<TItem>(TItem item) => Enumerable.Repeat(item, 1);
+        public static IEnumerable<TItem> Return<TItem>(TItem item)
+            where TItem : notnull
```

Comment:
Why the new `notnull` constraint?

Reply:
We discourage null values in our generators

Reply:
But isn't this a breaking change?

Reply:
Hah, good question. And I asked myself the same: 
* I did not need to change PublicAPI.Shipped, so it should be a compatible change, right?
* Obviously if you try to Return a nullable type in a nullable context it probably is incompatible?

I remove the constraint for now, but I'd really like to change this implementation which mandates the notnull constraint, since Generate does.

```cs
        public static IEnumerable<TItem> Return<TItem>(TItem item)
            where TItem : notnull
            => Generate(item, s => Option<TItem>.None());
```

Reply:
Removed constraint in 3855728b65b66a4fa3da577bf37ec0af643d83f0

Reply:
Yeah I think the Public API analyzer doesn't support the `notnull` constraint yet 🤷🏼

Reply:
Suggested change for 3.0 #260